forge/db/controllers/ProjectTemplate.js

```diff
@@ -198,6 +198,16 @@ module.exports = {
         if (typeof result.httpNodeAuth?.pass === 'string' && result.httpNodeAuth.pass.length > 0) {
             result.httpNodeAuth.pass = hash(result.httpNodeAuth.pass)
         }
+        if (result.apiMaxLength) {
+            if (!/^[0-9]+[km]?$/.test(result.apiMaxLength)) {
```

Comment:
Should we consider an upper limit?

Reply:
We can't set a sensible upper limit, we don't know how much memory the NR instance has or what might get dropped by any reverse proxy